question4/m1.c

```diff
@@ -0,0 +1,38 @@
+#include <stdio.h>
+
+
+struct{
+ 
+   
+    int score;  //成绩
+}class[] = {
+    {-1},
+    {-2},
+    {-3},
+    {-4},
+    {-5},
+    {0},
+    {1},
+    {2},
+    {3},
+    {4},
+    {5}
+};
+int main(){
+    int i;
+    int y;
+    int a, b, c;
+    printf("Please Enter Value of a:");
+    scanf("%d", &a);
+    printf("Please Enter Value of b:");
+    scanf("%d", &b);
+    printf("Please Enter Value of c:");
+    scanf("%d", &c);
+    for(i=0; i<11;){
```

Comment:
generally, it's better to not use a magic number and use a #define or a const global.